src/styles/blocks/stars.scss

```
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    color: $star-active-color;
+    margin-right: 4px;
+  }
+
+  &--5 &__star,
+  &--4 &__star:not(:last-child),
+  &--1 &__star:first-child,
+  &--2 &__star:nth-child(-n+2),
+  &--3 &__star:nth-child(-n+3) {
```

better to use consistent selectors

```suggestion
  &--1 &__star:nth-child(-n+1),
  &--2 &__star:nth-child(-n+2),
  &--3 &__star:nth-child(-n+3),
  &--4 &__star:nth-child(-n+4),
  &--5 &__star:nth-child(-n+5), {
```